src/client/rendering/game-renderer.ts

```diff
@@ -0,0 +1,55 @@
+import { autoDetectRenderer, Renderer, Container } from 'pixi.js'
+import Sprite from './sprite'
+
+export default class GameRenderer {
+	private static _instance: GameRenderer
+	static get instance() {
+		if(!GameRenderer._instance) {
+			GameRenderer._instance = new GameRenderer()
+		}
+		return GameRenderer._instance
+	}
```

Comment:
What do y'all think of this singleton pattern? Access is then `GameRenderer.instance.myFunction()` instead of `GameRenderer.GetInstance().myFunction()`.